### PR TITLE
renamed text job to phrases for easier comprehension

### PR DESCRIPTION
**Pull request by mackenza**
when I first looked at Kitto I looked at the text widget and didn't quite grok the relationship between a widget, a job and the dashboard. Part of this was because the job was named `text` as was the widget (well `Text` I guess technically).

I simply renamed `text.exs` to `phrases.exs` and changed the installer template so now the text widget on the sample dashboard is similar to the other widgets where the widget name is different than the job name (`data-source="phrases" in the dashboard).

It's a small change but I think could help.

### CODE REVIEW

**Comment by mackenza** [comment]
some more background can be found on Gitter where @davejlong and I discussed this topic. Thanks Dave!

**Comment by mackenza** [comment]
@zorbash if you want me to clean this up with a new PR let me know. I tested my initial changes but it was working because I hadn't changed the dashboard and job to match. Fixed now.

File: installer/templates/new/jobs/phrases.exs [hunk 1]
```diff
@@ -3,5 +3,5 @@ use Kitto.Job.DSL
 job :text, every: {4, :seconds} do
```
Comment (zorbash) [inline installer/templates/new/jobs/phrases.exs]: Rename the job to `:phrases` and remove the 1st argument which isn't required anymore from broadcast.

```elixir
job :phrases, every: {4, :seconds} do
  # ...
end
```

Comment (mackenza) [reply installer/templates/new/jobs/phrases.exs]: done

**Comment by zorbash** [comment]
Merged as 09bc604c3348467ca46c05df4163e8c8c2a57242
Thanks @mackenza

**Pull request closed without merging**